easybuild/easyconfigs/n/ncregrid/ncregrid-1.4b-goolf-1.4.10.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'ncregrid'
+version = '1.4b'
+
+homepage = 'http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/'
+description = """ncregrid is a tool (written in FORTRAN-90) for data transfer of gridded 2- and 3-dimensional (spatial) geophysical/geochemical scalar fields between grids of different resolutions"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'usempi': True, 'pic': True}
+
+source_urls = ['http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/']
+sources = [('%(name)s.tar.Z', 'tar -xzf %s')]
```

Comment:
since this source tarball is not versioned, please add a checksum

and specifying `tar -xzf` is no longer needed, since the framework now knows about `.tar.Z` (in develop, will be included in EB 2.4.0)